src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -76,12 +86,12 @@ def addon_selected?(_addon)
       # @return [Yast::Term] the main UI dialog term
       def content
         VBox(
-          VStretch(),
           Left(Heading(heading)),
+          Left(CheckBox(Id(:filter_beta), Opt(:notify),
```

Comment:
I think we should show this all the time so that the user learns and knows where it is. Instead of "Filter out Beta Products" I would change the wording/logic to "Show Beta Products (if available)" and no pre-select the checkbox. A user can then show the beta products, if desired, by selecting the checkbox.